### PR TITLE
Employer page

### PR DESCRIPTION
**Pull request by camillaume**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/CardEmployerInfo/index.jsx [hunk 1]
```diff
@@ -2,21 +2,102 @@ import agenda from "@assets/imgProfile/agenda.svg";
...
+    companyName: "",
+    companyStatus: "",
+    avatar: "",
+    banner: "",
+    descriptionShort: "",
+    status: "",
+    numberSiret: "",
+  });
+  const [modalIsOpen, setIsOpen] = useState(false);
+  const [modalIsOpen2, setIsOpen2] = useState(false);
```
Comment (LoicBrassart) [inline frontend/src/components/CardEmployerInfo/index.jsx:28]: I think you could come up with better variable names

File: frontend/src/components/CardEmployerInfo/index.jsx [hunk 2]
```diff
@@ -32,15 +113,17 @@ function CardEmployerInfo() {
...
+    content: {
+      top: "50%",
+      left: "50%",
+      right: "auto",
+      bottom: "auto",
+      marginRight: "-50%",
+      transform: "translate(-50%, -50%)",
+      lineHeight: "2rem",
+    },
+  };
```
Comment (LoicBrassart) [inline frontend/src/components/CardEmployerInfo/index.jsx:126]: Do you really think that's a good idea ?

**Review by LoicBrassart — APPROVED** [review]
Low quality code spotted, you know better than that.
Merge at your own risk :shrug:

**Pull request merged**